src/git.rs

```diff
@@ -103,8 +103,12 @@ fn find_last_commit(repo: &Repository) -> Result<Commit> {
 /// The method uses the default username and email address found for the
 /// repository. Defaults to the globally configured when needed.
 pub fn commit_to_repo(msg: &str, repository: impl AsRef<Path>) -> Result<Oid> {
-    let repo =
-        Repository::open(repository.as_ref().as_os_str()).expect("Failed to open git repository");
+    let repo = Repository::open_ext(
+        repository.as_ref().as_os_str(),
+        RepositoryOpenFlags::empty(),
+        vec![OsStr::new("")],
```

Comment:
Might be worth adding an option to prevent to go up here later on, but I'll still merge it.